core/src/main/java/jenkins/security/ApiTokenProperty.java

```diff
@@ -175,6 +175,7 @@ public boolean hasLegacyToken() {
 
     @NonNull
     @Restricted(NoExternalUse.class)
+    @SuppressFBWarnings(value = "UNSAFE_HASH_EQUALS", justification = "Only used to generate new token.")
```

Comment:
```suggestion
    @SuppressFBWarnings(value = "UNSAFE_HASH_EQUALS", justification = "Used to prevent use of pre-2013 API tokens, then returning the API token value")
```